### PR TITLE
Make grid in test-deck have larger number of cells.

### PR DESCRIPTION
**Pull request by atgeirr**
This makes some tests succeed that use this deck only for its properties and not its grid: if the grid in the deck has fewer cells than the grid that is used for the test we will fail in the SATNUM mapping.

This fixes the test_equil.cpp failure.

### CODE REVIEW

File: tests/capillary.DATA [hunk 1]
```diff
@@ -11,7 +11,7 @@ OIL
 GAS
 
 DIMENS
-3 3 3 /
+40 40 40 /
```
Comment (bska) [inline tests/capillary.DATA]: Twenty-seven cells are too few, as demonstrated by [`test_equil.cpp:CapillaryInversion`](https://github.com/OPM/opm-core/blob/master/tests/test_equil.cpp#L366-L369), but do you really need to go all the way to 64,000 cells for this?

Do keep in mind that some property arrays are created automatically behind the scenes if not provided in the deck and I don't think we need the to create such big arrays.  I'd suggest anything in the 5-by-5-by-5 to 10-by-10-by-10 region.

Comment (atgeirr) [inline tests/capillary.DATA]: I just wanted to minimise the odds of having this bite us again... The choice of 40 was due to one of the test grids actually having 40 cells in the k-direction (but 1 x 1 x 40 not 40^3...). It did not seem to ruin the speed of the test, so I think it does not hurt.

Comment (bska) [inline tests/capillary.DATA]: > The choice of 40 was due to one of the test grids actually having 40 cells in the k-direction

I see.  I hope you're not suggesting that we extend this to 100-by-100-by-100 if we one day create a test model that uses 100 cells in one direction.

Comment (atgeirr) [inline tests/capillary.DATA]: I might :-)

However then the tests might be slowed enough to be troublesome. I'm fine with the present size.

Comment (bska) [inline tests/capillary.DATA]: > I'm fine with the present size.

Very well, then.  I'll merge into master.

**Pull request merged**